apps/site/lib/site/trip_plan/related_link.ex

```diff
@@ -123,16 +123,8 @@ defmodule Site.TripPlan.RelatedLink do
     new(["View ", text, " fare information"], url)
   end
 
-  defp fare_link_text(:commuter_rail) do
-    "commuter rail"
-  end
-
-  defp fare_link_text(:ferry) do
-    "ferry"
-  end
-
-  defp fare_link_text(_) do
-    "bus/subway"
+  defp fare_link_text(type) when type in [:commuter_rail, :ferry, :bus, :subway] do
```

Comment:
Do we even need the `when` clause at this point?

Reply:
This was me being paranoid about whether a different type somehow snuck in here, but it looks like that won't happen given the [mode types coming in](https://github.com/mbta/dotcom/blob/383405b80994b0f6a347f390360cc63c3b029b29/apps/trip_plan/lib/trip_plan/api/open_trip_planner/parser.ex#L14) through OTP... can remove this